README.md

```diff
@@ -262,12 +262,17 @@ Framework, you can add a `shell.nix` based on this template:
 
 ```.nix
 # shell.nix
+{ pkgs ? import <nixpkgs> {} }:
+
+with pkgs;
 let
   kframework = import ./path/to/k {};
   inherit (kframework) mkShell;
```

Comment:
I also noticed this template might fail because `clang` is mentioned in `buildInputs` but it is not inherited from the `kframework` Nix package.

```suggestion
  inherit (kframework) mkShell clang;
```